test/celeritas/field/FieldPropagator.test.cc

```diff
@@ -538,7 +541,7 @@ TEST_F(TwoBoxTest, electron_small_step)
         EXPECT_FALSE(result.boundary);
         EXPECT_FALSE(geo.is_on_boundary());
         EXPECT_VEC_SOFT_EQ(Real3({5 + delta, 0, 0}), geo.pos());
-        EXPECT_VEC_SOFT_EQ(Real3({1, delta, 0}), geo.dir());
+        EXPECT_VEC_SOFT_EQ(Real3({1, 3 * delta, 0}), geo.dir());
```

Comment:
Same here. Why is the deviation in the y-direction getting larger?

Reply:
Same answer :)